data-es6.js

```diff
@@ -18316,7 +18319,7 @@ exports.tests = [
         var desc = Object.getOwnPropertyDescriptor(Object.prototype,"__proto__");
         var A = function(){};
 
-        return (desc
+        return (desc !== undefined
```

Comment:
this change makes it less strict; `desc` must be an object, all of which are truthy